Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.py

```diff
@@ -309,24 +316,81 @@ def test_function():
         return return_error(str(e))
 
 
+def fetch_incidents(last_run: dict = None):
+    dParams = demisto.params()
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get('time') if isinstance(last_run, dict) else None
+
+    # Handle first time fetch, fetch incidents retroactively
+    if last_fetch is None:
+        today = datetime.today()
+        delta = today - timedelta(days=50)
+        last_fetch = date_to_timestamp(delta)
+
+    incidents: list = []
+    dArgs = {}
+    dArgs['roleArn'] = dParams.get('roleArn')
+    dArgs['region'] = dParams.get('region')
+    dArgs['roleSessionName'] = dParams.get('roleSessionName')
+    dArgs['roleSessionDuration'] = dParams.get('roleSessionDuration')
+    dArgs['analyzerArn'] = dParams.get('analyzerArn')
+    dArgs['maxResults'] = 25
```

Comment:
It's better to initialize params in the `main` function

Reply:
It seems this integration does not have a main function, thoughts on leaving this parameter initialization in the fetch function until the entire integration is rewritten to comply with all our current standards?

Reply:
sounds good to me